### ISSUE TITLE
Success Criterion 1.4.12 - Text Spacing - Level AA

### ISSUE THREAD

**Issue by JJdeGroot** [w3c/matf#5]
Discussion:

<details>
<summary>8 May 2024</summary>

[Source](https://www.w3.org/2024/05/08-matf-minutes.html#t02)

## 1.4.12 Text Spacing

[JJ moves to relevant row in screenshare]

JJ: Joe is saying might be a no, there are bookmarklets for testing on web

JJ: to 'applicable to mobile native', Joe also added 'no'

quintinb: with this, what we probably can do, we strongly recommend that you respect user text settings or just user settings

<Joe_Humbert> I agree with quintinb that it could be added in the future.

<quintinb> Yeah not in stock Android for now

<quintinb> Oooh what do we think about apps that force a specific font?

julianmka: I tell designers there is no such thing as a canonical view of an app… supporting capabilities is important

<julianmka> I'm still testing but Samsung's OneUI font swapping only applies to apps that use the designated system font.

JJ: I think in the EN 301 549 there is a mention of that software should implement text properties of the software system or something

<quintinb> Is TalkBack on?

Double_a: font sizes are difficult to manage… we recently did a survey … if the accessibility features to make content bold are turned on, and the content was already bold, it will make it unreadable

Double_a: we observed this with nearsighted users

<julianmka> We can't test every potential combo of dubious font settings, but people should have the decision to use them if they want to.

<Karla> +1

JJ: so would say this SC is not applicable but we could add some best practices

Double_a: I did write a blog post on the font specific stuff, would be happy to share it

<Joe_Humbert> can developers force the use of the font used in the application over the OS font?

<quintinb> yes they can in Android

<quintinb> I was going to ask about this

JJ: yes they can

JJ: as far as I know there is no way for users to override

quintinb: if the user could set their font, should we require that apps respect the user setting?

JJ: would also be interesting to look at the EN standard

<quintinb> I could see this being useful for folks, but also makes life heck for designers

<AlainVagner> the EN has a specific criterion on this

<AlainVagner> 11.7

<AlainVagner> user prefs

JJ: shares 11.7 User Preferences of EN 301 549

<Joe_Humbert> font type possibly serif vs sans serif?

<JJ> Hidde: chapter 11 of EN 301 549 is likely to get expanded

JJ: we'll discuss 1.4.13 and 2.4.1 next time

<quintinb> Thanks JJ and Hidde!

</details>

**1. Applicability to Mobile Native:**
- **JJ:** Noted Joe's input that this criterion might not be applicable to mobile native apps.
- **QuintinB:** Recommended respecting user text settings in apps.
- **Joe_Humbert:** Agreed with the potential for future inclusion, though it's not in stock Android currently.

**2. Font Usage in Apps:**
- Apps forcing specific fonts can impact accessibility.
- **Julianmka:** Emphasized the importance of supporting user capabilities over a canonical view of the app.
- **JJ:** Mentioned EN 301 549's guidance on implementing text properties in software.

**3. Managing Font Sizes:**
- **Double_a:** Shared survey findings on difficulties managing font sizes, especially when accessibility features like bold text are turned on, which can make content unreadable.
- **Consensus:** Users should have control over font settings, but testing every combination of settings is impractical.

**4. Developer Control Over Fonts:**
- **Joe_Humbert:** Asked about developers forcing app-specific fonts over OS fonts.
- **JJ & Others:** Confirmed developers can force font usage in Android, and there's no way for users to override this.

**5. Respecting User Font Settings:**
- **QuintinB:** Suggested apps should respect user-set fonts if the OS allows it.
- **JJ:** Proposed looking into the EN 301 549 standard for more guidance.

**6. EN 301 549 Standard:**
- **JJ:** Shared section 11.7 User Preferences from EN 301 549.
- **Hidde:** Mentioned the potential expansion of chapter 11 in EN 301 549.

**Comment by jamieherrera** [comment]
We should still include this in the guidance document but with a brief Note why this is n/a

**Comment by JJdeGroot** [comment]
@jamieherrera yes I was just going through the list, and we should add this to our draft.
@julianmka , I've assigned you for drafting, is that ok?
